one_fm/utils.py

```diff
@@ -961,10 +975,8 @@ def get_new_leave_allocated_for_annual_paid_leave(allocation, leave_type):
     new_leaves_allocated = 0
 
     # Fetch employee annual leave allocation from employee
-    employee_annual_leave = frappe.db.get_value('Employee', allocation.employee, 'annual_leave_balance')
```

Comment:
Please you removed a block of code here, is this intentional ?

Reply:
Yes

Reply:
I don't remove it. I just wrap it in try and catch

Reply:
please, if you observe it below, the block was removed, and replaced with a single query